products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -1,15 +1,207 @@
-# Global Documentation Search
+# Documentation Search MVP
 
-Author: Bill Fienberg
-Last Updated: 2020-01-23
-Reviewers: TBD
+**Author(s):** Bill Fienberg <bill.fienberg@oddball.io>
 
-## Objective
+**Last Updated:** 2020-02-13
 
-## Background
+**Status:** **Draft**
+
+**Approvers:** Andrew G., Megan K., Rian F., Wyatt W.
 
 ## Overview
 
-## Detailed Design
+### Objective
+
+Sign up for Algolia, add a search input and Algolia's DocSearch JS snippet to the root page of `va.gov-team`'s GitHub Pages site, and configure CircleCI to run's Algolia's scraper to crawl our docs once per day.
+
+### Background
+
+The documentation we have is spread across many different locations (public repos, private repos, GitHub pages, handbooks, etc). There currently isn't a way for VFS/VSP users to search multiple documentation sources with a single query.
+
+### High Level Design
+
+## Specifics
+
+### Detailed Design
+
+For a multi-repo search MVP, we are adding an HTML text input and [Algolia's DocSearch JS snippet](https://github.com/algolia/docsearch) to the [`va.gov-team` GitHub Pages site](https://department-of-veterans-affairs.github.io/va.gov-team/). This will use [Algolia's Search API](https://www.algolia.com/products/search/) to query Algolia's hosted database (https://www.algolia.com/pricing/), which will be populated by [Algolia's scraper](https://github.com/algolia/docsearch-scraper) using a [configuration we supply](docs/docsearch-scraper-config.json).
```

Comment:
What repositories would we end up searching? The MVP lists this repo, but what others are you targeting?

Reply:
We will crawl and scrape the following public documentation sources:

- [`va.gov-team` repo](https://github.com/department-of-veterans-affairs/va.gov-team)
- [`veteran-facing-services-tools` repo](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/)
- [`vets-contrib` repo](https://github.com/department-of-veterans-affairs/vets-contrib/)
- [design.va.gov](https://design.va.gov/)
- [developer.va.gov](https://developer.va.gov/)

The list was added to the design doc in https://github.com/department-of-veterans-affairs/va.gov-team/pull/5248/commits/8ba0b54f413ef814861cfbc06bb0f1b1f7b36efc